docs/contributing.md

```diff
@@ -2,37 +2,92 @@
 
 ## Mailing list
 
-The main communication tool we use is a mailing list:
+The main communication tool in use is the Yocto Project mailing list:
+
 * <yocto@yoctoproject.org>
 * <https://lists.yoctoproject.org/listinfo/yocto>
 
-Feel free to ask any kind of questions but always prepend your email subject
-with "[meta-raspberrypi]". This is because we use the 'yocto' mailing list and
-not a perticular 'meta-raspberrypi' mailing list.
+Feel free to ask any kind of questions but please always prepend your email
+subject with `[meta-raspberrypi]` as this is the global *Yocto* mailing
+list and not a dedicated *meta-raspberrypi* mailing list.
+
+## Formatting patches
+
+First and foremost, all of the contributions to the layer must be compliant
+with the standard openembedded patch guidelines:
+
+* <http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines>
+
+In summary, your commit log messages should be formatted as follows:
+
+    <layer-component>: <short summary of changes>
+
+    <more detailed description of changes and considerations>
+
+    Signed-off-by: Your Name <yourname@youremail.com>
+
+The `<layer-component>` is the layer component name that your changes affect.
+It is important that you choose it correctly. A simple guide for selecting a
+a good component name is the following:
+
+* For changes that affect the *layer itself* such as adding new recipes or
```

Comment:
this is not exactly the practice. when adding new recipe you say
```newrecipe: Add version 1.0.0```
so remove that, when changing multiple recipes use
```recipe1,recipe2,recipe3...: Change ...```

Reply:
Thanks for the multiple and new recipes suggestion, will reflect.